src/rgw/rgw_admin.cc

```diff
@@ -4249,10 +4249,33 @@ int main(int argc, const char **argv)
     bool need_cache = readonly_ops_list.find(opt_cmd) == readonly_ops_list.end();
     bool need_gc = (gc_ops_list.find(opt_cmd) != gc_ops_list.end()) && !bypass_gc;
 
+    std::string rgw_store = "rados";
+    const auto& config_store = g_conf().get_val<std::string>("rgw_backend_store");
+    #ifdef WITH_RADOSGW_DBSTORE
+    if (config_store == "dbstore") {
+      rgw_store = "dbstore";
+    }
+    #endif
+
+    #ifdef WITH_RADOSGW_MOTR
+    if (config_store == "motr") {
+      rgw_store = "motr";
+    }
+    #endif
+
+    #ifdef WITH_RADOSGW_DAOS
+    if (config_store == "daos") {
+      rgw_store = "daos";
+    }
+    #endif
+
+    lsubdout(cct, rgw, 1) << "RGW CONF BACKEND STORE = " << config_store << dendl;
+    lsubdout(cct, rgw, 1) << "RGW BACKEND STORE = " << rgw_store << dendl;
+
     if (raw_storage_op) {
-      store = StoreManager::get_raw_storage(dpp(), g_ceph_context, "rados");
+      store = StoreManager::get_raw_storage(dpp(), g_ceph_context, rgw_store);
     } else {
-      store = StoreManager::get_storage(dpp(), g_ceph_context, "rados", false, false, false,
+      store = StoreManager::get_storage(dpp(), g_ceph_context, rgw_store, false, false, false,
```

Comment:
This allows `radosgw-admin` to create, update and delete users. Another PR (#45247 ) includes [a similar change](https://github.com/ceph/ceph/pull/45247/files#diff-f3b81fb31115c2616085efcfd6a8daed67a94d1041a03af5de48cc572978f547). We are working together to make sure that  `radosgw-admin` works well with daos and motr.